### PR TITLE
Updated the losses.py for the reduction loss function

### PR DESCRIPTION
**Pull request by RenuPatelGoogle**
"reduction=losses_utils.ReductionV2.AUTO" loss function does not exist. 

Hence, updated the "reduction=losses_utils.ReductionV2.AUTO" with correct working reduction loss function which is "reduction=tf.keras.losses.Reduction.AUTO".

### CODE REVIEW

**Comment by sachinprasadhs** [comment]
`Reductionv2` has `AUTO` mode here https://github.com/keras-team/tf-keras/blob/master/tf_keras/utils/losses_utils.py#L29
